public/app/features/search/components/SearchResults.tsx

```diff
@@ -25,43 +28,62 @@ export const SearchResults: FC<Props> = ({
   onToggleChecked,
   onToggleSection,
   results,
+  wrapperRef,
   layout,
 }) => {
   const theme = useTheme();
   const styles = getSectionStyles(theme);
+  const listHeight = useListHeight(wrapperRef?.current?.offsetTop);
```

Comment:
Do we need this, in other places we have used AutoSizer to measure element height or width.

Reply:
`AutoSizer`didn't work properly when virtualizing nested items, but sine we don't do that now Ill give it another try.